jsk_pcl_ros/src/extract_indices_nodelet.cpp

```diff
@@ -92,6 +92,16 @@ namespace jsk_pcl_ros
     pcl_conversions::toPCL(*cloud_msg, *input);
     pcl::PointIndices::Ptr indices (new pcl::PointIndices ());
     pcl_conversions::toPCL(*indices_msg, *indices);
+    
+    // check if input size is bigger than indices size 
+    uint8_t data_size = static_cast<uint8_t>(input->data.size());
+    uint32_t point_step_size = static_cast<uint32_t>(input->point_step);
+    uint8_t cloud_size = data_size / point_step_size;
+    uint8_t indices_size = static_cast<uint8_t>(indices->indices.size());
+    if (cloud_size <= indices_size){
```

Comment:
```suggestion
    if (cloud_size < indices_size){
```

Reply:
is this really correct?

Reply:
Mine is wrong since when it has equality, extract whole point cloud.

Reply:
good